portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,100 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Portfolio | Andrew Thankson</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body class="container">
+    
+
+<div class="jumbotron">
+        <div class="container">
+        <p>
+        <h1 class="display-3">Andrew Thankson</h1>
+        <img class="andrew-img" src="images/andrew.png" width="100px" height="150px" alt="picture of andrew thankson">
+        </p>
+        <h2>A little about me</h2>
+        <p>Hi all, I'm Andrew and I'm glad you're here! I'm currently a Sophomore Computer Science Major and Philosophy minor at the
```

Comment:
Google's HTML style guide suggests wrapping long lines to make it easier to read: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

You might consider adding some linebreaks for these long lines?

Reply:
Awesome! Thanks for the feedback!